blueprints/reset.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+from dependency_injector.wiring import Provide, inject
+from flask import Blueprint, Response, request
+from flask.views import MethodView
+
+import demo
+from containers import Container
+from repositories import UserRepository
+
+from .util import class_route
+
+blp = Blueprint('Reset database', __name__)
+
+
+@class_route(blp, '/api/v1/reset/user')
+class ResetDB(MethodView):
+    init_every_request = False
```

Comment:
The flask documentation says
> By default, a new instance of the view class is created every time a request is handled. This means that it is safe to write other data to self during the request, since the next request will not see it, unlike other forms of global state.
>
> However, if your view class needs to do a lot of complex initialization, doing it for every request is unnecessary and can be inefficient. To avoid this, set [View.init_every_request](https://flask.palletsprojects.com/en/3.0.x/api/#flask.views.View.init_every_request) to False, which will only create one instance of the class and use it for every request. In this case, writing to self is not safe. If you need to store data during the request, use [g](https://flask.palletsprojects.com/en/3.0.x/api/#flask.g) instead.